sanitizers/BUILD.bazel

```diff
@@ -7,9 +7,9 @@ java_library(
 )
 
 java_library(
-    name = "offline_only_sanitizers",
```

Comment:
This is also dead code from before sanitizers were injected into bootstrap classloader on Android

Reply:
... the offline_only_sanitizers is dead code, but the new android_sanitizers is needed